.github/workflows/prune-images.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     steps:
       - name: Set default inputs when triggered via schedule
         run: |
-          echo "ENVIRONMENT=${{ inputs.environment || 'test' }}" >> $GITHUB_ENV
+          echo "ENVIRONMENT=${{ inputs.environment || 'dev' }}" >> $GITHUB_ENV
```

Comment:
Thanks for making the change. Please also change the default days for the scheduled job at line 45.
```
echo "MIN_TAGS=${{ inputs.minTags || '2' }}" >> $GITHUB_ENV
```

Reply:
Done!